rocon_remocon/src/rocon_remocon/interactive_client.py

```diff
@@ -104,9 +104,21 @@ def _connect(self, rocon_master_name="", ros_master_uri="http://localhost:11311"
         # for the remocon-role manager interaction anyway:
         rospy.init_node(self.name, disable_signals=True)
         try:
-            get_interactions_service_name = rocon_python_comms.find_service('rocon_interaction_msgs/GetInteractions', timeout=rospy.rostime.Duration(5.0), unique=True)
-            get_roles_service_name = rocon_python_comms.find_service('rocon_interaction_msgs/GetRoles', timeout=rospy.rostime.Duration(5.0), unique=True)
-            request_interaction_service_name = rocon_python_comms.find_service('rocon_interaction_msgs/RequestInteraction', timeout=rospy.rostime.Duration(5.0), unique=True)
+            console.logdebug("Interactive Client : Get interactions service Handle")
+            interactions_namespace = rocon_python_comms.find_namespace('get_interactions', 'rocon_interaction_msgs/GetInteractions')
+
+            remocon_services = {}
+            remocon_services['get_interactions'] = interactions_namespace + '/' + 'get_interactions'
+            remocon_services['get_roles'] = interactions_namespace + '/' + 'get_roles'
+            remocon_services['request_interaction'] = interactions_namespace + '/' + 'request_interaction'
+
+            for service_name in remocon_services.keys():
+                if not rocon_python_comms.is_valid_service(remocon_services[service_name]):
+                    raise rocon_python_comms.NotFoundException("'%s' service is not validated" % service_name)
+        except rocon_python_comms.MultipleFoundException as e:
+            message = "find multiple interactions' publications and services [%s]. Please check interactions remap rules" % str(e)
```

Comment:
@jihoonl made a good comment too....this should do more than just warn...it should gracefully exit with the error message when it fails.